logging.properties

```diff
@@ -0,0 +1,6 @@
+.level = FINE
+handlers = java.util.logging.ConsoleHandler
```

Comment:
Unfortunately, the output does not go to Gradle's build scan console log...

https://scans.gradle.com/s/o45ypt3zbpnic/console-log